Release 1.1.4

kevinkirkup opened the pull request.

Commit:
Merge branch 'release/1.1.2' into develop

Commit:
Copy to Nexus for only on master and develop

Commit:
Update to golang 1.13 (#56)

Commit:
Adjust maximum ASN value based on 32bit architecture (#57)

Commit:
Merge branch 'hotfix/1.1.3' into develop

Commit:
Additional updates for golang 1.13 (#59)

Commit:
Terraform is using the CHANGELOG.md for identifying the version number (#60)

Updating to reflect what is in Github

Commit:
Update to Terraform 0.12.10 (#61)

* Update to Terraform 0.12.10

* Update Changelog for next release

Commit:
Ignore developer artifacts

Commit:
[PE-1694] Google Router and Interconnects need unique names (#62)

* Update to latest AzureRM release

* Make sure we are allocating unique names for google interconnects and routers

* Remove Template Provider

Commit:
Update Changelog

Commit:
Merge branch 'master' into develop

Try75 reviewed (approved):
Everything outside of the `vendors/` directory looks right.

kevinkirkup commented:
This PR includes #1 changes as well.

Commit:
Tools need to be vendored

Commit:
Merge pull request #2 from pureport/feature/fix-travis-ci-issues

Tools need to be vendored

Commit:
Fix lint error and add TF Provider linter as well

Commit:
Merge pull request #3 from pureport/feature/fix-travis-ci-issues

Fix lint error and add TF Provider linter as well

Commit:
Reduce the memory for the linter

Commit:
Merge pull request #4 from pureport/feature/fix-travis-ci-issues

Reduce the memory for the linter

Commit:
Increase the deadline since we are using less memory

Also use the default number of jobs, which should equal the number
of available cores.

Commit:
Merge pull request #5 from pureport/feature/fix-travis-ci-issues

Increase the deadline since we are using less memory
